client/js/components/content-slider.js

```diff
@@ -365,6 +364,14 @@ const contentSlider = (el, options) => {
   sliderElements.prevControl.addEventListener('click', prevSlide, true);
   sliderElements.nextControl.addEventListener('click', nextSlide, true);
 
+  nodeList(sliderElements.slideItems).forEach((item) => {
+    item.addEventListener('click', ({ target, currentTarget }) => {
+      if (!target.matches('img')) return;
```

Comment:
It feels like we should be adding the event listener to the element that we want no?
Either way, this works, but if we keep it, maybe make a note of why?